LongWarOfTheChosen/Src/LW_FactionBalance/Classes/X2Effect_BattlefieldAwarenessCooldown.uc

```diff
@@ -0,0 +1,13 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Effect_BattlefieldAwareness
+//  AUTHOR:  Grobobobo
+//  PURPOSE: A dummy effect That represents the cooldown of battlefield awareness
```

Comment:
Do you even need this? Can't you just use `X2Effect_Persistent` and set its `EffectName` when you attach it to the ability?

Reply:
That was the OG Idea, but I've created the effect when it didn't work to check if some things didn't work. Not sure if I should change back at this point.